release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2023,6 +2023,22 @@ As a workaround, use fully qualified names, such as `registry.redhat.io/ubi8/ubi
 +
 Optionally, you can update the image registry settings by link:https://docs.openshift.com/container-platform/4.9/openshift_images/image-configuration.html#images-configuration-shortname_image-configuration[adding registries that allow image short names]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011293[*BZ#2011293*])
 
+* The `oc adm release extract --tools` command fails when an image that is hosted in the local registry is specified. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1823143[BZ#1823143])
+
+* On an {product-title} single node configuration, pod creation times are over two times slower when using the real-time kernel (`kernel-rt`) than when using the non-real time kernel. When using `kernel-rt`, the slower pod creation times affect the maximum number of supported pods because recovery time is impacted after a node reboots.
++
+As a workaround, when you use `kernel-rt`, you can improve the recovery time by booting with the `rcupdate.rcu_normal_after_boot=0` kernel argument. This requires a real-time kernel version `kernel-rt-4.18.0-305.16.1.rt7.88.el8_4` or later. This known issue applies to {product-title} version 4.8.15 and later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975356[BZ#1975356])
```

Comment:
One extra thing, I think the BZ# need to be bolded, so you should apply [*BZ123456*] to make these bold.